### ISSUE TITLE
Could not get information from post...

### ISSUE THREAD

**Issue by rolaxbell** [timgrossmann/instagram-profilecrawl#10]
Everything works great. After the script runs it creates the json file and outputs the profile information. However, when it starts crawling individual posts, although I can see the driver scanning the correct post, it fails to grab any information relevant to any post.

To troubleshoot, I added some print statements in the extract_post_info function to see if it was grabbing info. It's able to perform well up to here: 

```
 if len(imgs) >= 2:
    img = imgs[1].get_attribute('src')
    print(img) #added print statement
```

After that, I tried adding some print statements here:

```
likes = likes.split(' ')
  
  print("likes is: ", likes) #my addition

  #count the names if there is no number displayed
  if len(likes) > 2:
    likes = len([word for word in likes if word not in ['and', 'like', 'this']])
    print(likes) #my addition
  else:
    likes = likes[0]
    likes = likes.replace(',', '').replace('.', '')
    likes = likes.replace('k', '00')
```

But nothing prints out. I assume, this must be related to the issue of the function not returning anything and thus leading to the `    except NoSuchElementException:
      print('- Could not get information from post: ' + link)`

What do you think could be wrong? I don't want to alter the code too much since I'm not particularly familiar with selenium.

Any help would awesome! 

Thanks!

**Comment by timgrossmann** [comment]
@rolaxbell Thank you for noticing this, I will look into this tomorrow or at the start of next week

**Comment by tcvieira** [comment]
I think my pull request fixed this #11
